src/mobile/src/navigation/Navigation.jsx

```diff
@@ -0,0 +1,83 @@
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { View } from 'react-native';
+import { MaterialIcons, Entypo, EvilIcons } from '@expo/vector-icons';
+import SearchScreen from '../screens/SearchScreen';
+import HomeScreen from '../screens/HomeScreen';
+import SignUpScreen from '../screens/SignUpScreen';
+import ContactScreen from '../screens/ContactScreen';
+
+const Tab = createBottomTabNavigator();
+
+const Navigation = () => {
+  return (
+    <Tab.Navigator
+      screenOptions={{
+        tabBarShowLabel: false,
+        tabBarStyle: [
+          {
+            display: 'flex',
+          },
+          null,
+        ],
+      }}
+    >
+      <Tab.Screen
+        name="Search"
+        component={SearchScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <EvilIcons name="search" size={24} color="#121D59" />
+            </View>
+          ),
+        }}
+      />
+      <Tab.Screen
+        name="Home"
+        component={HomeScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <Entypo name="home" size={24} color="#121D59" />
+            </View>
+          ),
+        }}
+      />
+      <Tab.Screen
+        name="Contact"
+        component={ContactScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <MaterialIcons name="contact-support" size={24} color="#121D59" />
+            </View>
+          ),
+        }}
+      />
+      <Tab.Screen
+        name="SignUp"
+        component={SignUpScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <MaterialIcons name="exit-to-app" size={24} color="#121D59" />
+            </View>
+          ),
+        }}
+      />
+      <Tab.Screen
+        name="theme"
```

Comment:
Use the same name convention

```suggestion
        name="Theme"
```